src/redux/reducers/loginReducer.js

```diff
@@ -0,0 +1,24 @@
+import ActionTypes from '../../constants/ActionTypes'
```

Comment:
the filepath of `src/redux/` seems kinda weird. thinking about moving all the subfolders of `redux` 1 level higher, so the structure is flat. it'll more closely match typical Redux file structures i've seen online. thoughts?